### PR TITLE
Add custom svg components for lock and flash decorators

### PR DESCRIPTION
**Pull request by SlimaneAmar**
**Please check if the PR fulfills these requirements** *(please use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes)*
- [ x] The commit message follows our guidelines

**Does this PR already have an issue describing the problem ?** *If so, link to this issue using `'#XXX'` and skip the rest*
No

**What kind of change does this PR introduce?** *(Bug fix, feature, docs update, ...)*
Feature

**What is the current behavior?** *(You can also link to an open issue here)*
Lock and flash decorators use the default svg components

**What is the new behavior (if this is a feature change)?**
Lock and flash decorators use the custom svg components

### CODE REVIEW

File: src/main/resources/GridsuiteLibrary/lock_black_24dp.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" height="24px" viewBox="0 0 24 24" width="24px" fill="#000000"><g fill="none"><path d="M0 0h24v24H0V0z"/><path d="M0 0h24v24H0V0z" opacity=".87"/></g><path d="M18 8h-1V6c0-2.76-2.24-5-5-5S7 3.24 7 6v2H6c-1.1 0-2 .9-2 2v10c0 1.1.9 2 2 2h12c1.1 0 2-.9 2-2V10c0-1.1-.9-2-2-2zM9 6c0-1.66 1.34-3 3-3s3 1.34 3 3v2H9V6zm9 14H6V10h12v10zm-6-3c1.1 0 2-.9 2-2s-.9-2-2-2-2 .9-2 2 .9 2 2 2z"/></svg>
```
Comment (FranckLecuyer) [inline src/main/resources/GridsuiteLibrary/lock_black_24dp.svg:1]: Maybe add newline at end of file ?

File: src/main/resources/GridsuiteLibrary/bolt_black_24dp.svg [hunk 2]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" enable-background="new 0 0 24 24" height="24px" viewBox="0 0 24 24" width="24px" fill="#000000"><g><rect fill="none" height="24" width="24"/></g><g><path d="M11,21h-1l1-7H7.5c-0.88,0-0.33-0.75-0.31-0.78C8.48,10.94,10.42,7.54,13.01,3h1l-1,7h3.51c0.4,0,0.62,0.19,0.4,0.66 C12.97,17.55,11,21,11,21z"/></g></svg>
```
Comment (FranckLecuyer) [inline src/main/resources/GridsuiteLibrary/bolt_black_24dp.svg:1]: Maybe add newline at end of file ?

**Review by FranckLecuyer — CHANGES_REQUESTED** [review]

File: src/main/resources/GridsuiteLibrary/bolt_black_24dp.svg (hunk 2 above)
Comment (SlimaneAmar) [reply src/main/resources/GridsuiteLibrary/bolt_black_24dp.svg]: Done

File: src/main/resources/GridsuiteLibrary/lock_black_24dp.svg (hunk 1 above)
Comment (SlimaneAmar) [reply src/main/resources/GridsuiteLibrary/lock_black_24dp.svg]: Done

**Review by geofjamg — APPROVED** [review]

**Pull request merged**